ios/brave-ios/Sources/Brave/Frontend/Settings/Debug/Brave Wallet/BraveWalletDebugMenu.swift

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import BraveUI
+import BraveWallet
+import Preferences
+import SwiftUI
+
+struct BraveWalletDebugMenu: View {
+
+  @ObservedObject var enableBitcoinTestnet = Preferences.Wallet.isBitcoinTestnetEnabled
+
+  var body: some View {
+    List {
+      Section {
+        Toggle("Enable Bitcoin Testnet", isOn: $enableBitcoinTestnet.value)
+          .toggleStyle(SwitchToggleStyle(tint: .accentColor))
+      }
+    }
+    .listBackgroundColor(Color(UIColor.braveGroupedBackground))
```

Comment:
Could we add a title to this view? I think just `Brave Wallet Debug` / `Wallet Debug` is fine, just for some context once the menu is opened.

And not a big deal since it's in QA / Debug menus but do we also want to hide it behind the Bitcoin feature flag? And since it's the only debug option in the menu we probably should just hide the entire `View Brave Wallet Debug Menu` row behind the feature flag as well 😅.

Reply:
added in [8193cb9](https://github.com/brave/brave-core/pull/22971/commits/8193cb9c02fb3fbabeac81084bb564414ca2c7e6)